### PR TITLE
[ios] add crosswatch64_32 target

### PR DESCRIPTION
**Pull request by lewurm**
work in progress, the resulting crosswatch64_32 aot compiler doesn't work yet.

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -4236,7 +4236,7 @@ case "$host" in
 		CPPFLAGS="$CPPFLAGS -D__ARM_EABI__"
 		;;
 	aarch64-*darwin*ilp32)
-		TARGET=ARM64;
+		TARGET=ARM6432
```
Comment (lewurm) [inline configure.ac]: Note that https://github.com/mono/mono/blob/972f29179dc11558902d89b55ae2f333c71a9aa9/configure.ac#L4442-L4445 already exists on master

**Comment by lewurm** [comment]
@monojenkins squash

**Comment by luhenry** [comment]
@monojenkins build Linux WebAssembly

File: tools/offsets-tool/MonoAotOffsetsDumper.cs [hunk 2]
```diff
@@ -188,7 +196,7 @@ public static void SetupOtherTargets()
             } else if (abi == "wasm32-unknown-unknown") {
                 Targets.Add(new Target {
                         Platform = TargetPlatform.WASM,
-                        Triple = "wasm32-wasm32-unknown-unknown",
+                        Triple = "wasm32-unknown-unknown",
```
Comment (lewurm) [inline tools/offsets-tool/MonoAotOffsetsDumper.cs]: @vargaz will that break anything?

**Comment by lewurm** [comment]
@monojenkins build Linux AArch64 Coop Suspend

**Comment by luhenry** [comment]
@monojenkins backport to 2019-02
@monojenkins backport to 2018-12
@monojenkins backport to 2018-10
@monojenkins backport to 2018-08-watchos

**Comment by luhenry** [comment]
@lewurm could you please make sure to backport to `2018-10` and `2018-08-watchos`?

**Pull request merged**